pkg/importer/datastream_ginkgo_test.go

```diff
@@ -150,12 +151,16 @@ var _ = Describe("Streaming Data Conversion", func() {
 				// Generate the expected data format from the random bytes
 				testSample, err := utils.FormatTestData(origFile, tmpTestDir, ff...)
 				Expect(err).NotTo(HaveOccurred(), "Error formatting test data.")
-				fmt.Fprintf(GinkgoWriter, "INFO: converted source file name is %q\n", testSample)
+				fmt.Fprintf(GinkgoWriter, "INFO: converted source file name is %q, in dir %q\n", testSample, tmpTestDir)
+
+				ts := createTestServer(filepath.Dir(testSample))
 
-				testEp := "file://" + testSample
 				testTarget := filepath.Join(tmpTestDir, common.ImporterWriteFile)
-				By(fmt.Sprintf("Importing %q to %q", testEp, testTarget))
-				err = importer.CopyImage(testTarget, testEp, "", "")
+				By(fmt.Sprintf("Importing %q to %q", ts.URL, testTarget))
+				copyDs, err := importer.NewDataStream(ts.URL+"/"+filepath.Base(testSample), "", "")
+				Expect(err).NotTo(HaveOccurred(), "Error reading source data.")
+				defer copyDs.Close()
```

Comment:
What is the reason to no longer test `CopyImage` here?

Reply:
Well the reason was that I needed to read some data from the stream or the stream would stop waiting for a reader, but I have since fixed the problem with that, so I should simply use CopyImage.